apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Logging/LoggingTest.cs

```diff
@@ -297,7 +298,7 @@ public class TimedBufferWarningLoggerTestApplication : LoggerTestApplication
         public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         {
             SetupRoutes(app);
-            var options = BufferOptions.TimedBuffer(TimeSpan.FromSeconds(20));
+            var options = BufferOptions.TimedBuffer(TimeSpan.FromSeconds(10));
```

Comment:
would it be better to manually flush at test completion rather than rely on a timer?

Reply:
This was an accidental change.  I reverted it.  Yes that's probably a better way to go about this though.  I'll give that a try later.